tools/licensescan/main.go

```diff
@@ -125,7 +129,14 @@ func RunLicenseScan(ctx context.Context, opts RunLicenseScanOptions) error {
 		modules = append(modules, module)
 
 		p := filepath.Join("modules", module.Name, module.Version+".yaml")
-		b, err := ioutil.ReadFile(p)
+		b, err := os.ReadFile(p)
+		if err != nil && os.IsNotExist(err) {
+			b2, err2 := modulesFS.ReadFile(p)
+			if err2 == nil {
+				b = b2
+				err = err2
+			}
```

Comment:
err2 != nil is being ignored and I believe that's intentional because err is what we care about in that case.

Reply:
Yes, it is deliberate (I think), but I think I'll take a second look and maybe log a `V(4)` message to make the code at least more self-explanatory.